pkgs/applications/video/kodi/addons/invidious/default.nix

```diff
@@ -21,12 +18,16 @@ buildKodiAddon rec {
     inputstreamhelper
   ];
 
+  prePatch = ''
+    substituteInPlace Makefile --replace 'zip -r' '${zip}/bin/zip -r'
```

Comment:
i looked at the Makefile and it takes source to create a zip file... but we don't use the zip file

i don't think we need the Makefile in this case, do we?

Reply:
When `fetchzip` is used the `Makefile` is gone from `src` :)